duckduckgo/iOS
# Optimizations to AppTP Manage Trackers View

Task/Issue URL: https://app.asana.com/0/1203881841413756/1204456266799291/f
Tech Design URL:
CC: @samsymons 

**Description**:
This PR adds optimizations to the Manage Trackers view to improve user experience when loading the view. Also optimizes toggling a tracker to not have to re-build the full tracker list.



**Steps to test this PR**:
1. Go to AppTP -> Manage Trackers. Previously there was a pause before the view opened. View should now open immediately and a brief spinner will show as the tracker list is built.
2. Toggle some trackers. Ensure they persist between view loads. Ensure trackers aren't blocked if they're allowed.



**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
